packages/jest-circus/src/index.js

```diff
@@ -54,13 +59,51 @@ const test = (testName: TestName, fn: TestFn, timeout?: number) => {
       `Invalid second argument, ${fn}. It must be a callback function.`,
     );
   }
-  return dispatch({fn, name: 'add_test', testName, timeout});
+  
+  const asyncError = new Error();
+  if (Error.captureStackTrace) {
+    Error.captureStackTrace(asyncError, test);
```

Comment:
why is stack captured here but not in the other ones?

Reply:
Mostly because I'm lazy.

`Error.captureStackTrace` removes the frame (function) from the second argument, so this in practice removes this function from the stack, meaning users don't get `circus` in their output.